cmd/snap/cmd_snap_op.go

```diff
@@ -282,9 +282,35 @@ func (x *cmdTry) Execute([]string) error {
 	return listSnaps([]string{name})
 }
 
+type cmdRollback struct {
+	Positional struct {
+		Snap string `positional-arg-name:"<snap>"`
+	} `positional-args:"yes"`
+}
+
+var shortRollbackHelp = i18n.G("Rollback the given snap to the previous version")
+var longRollbackHelp = i18n.G(`
+The rollback command will rollback the given snap to the previous version.
```

Comment:
```
The rollback command will revert the given snap to its state before the latest refresh.
This will make the previous revision in use active again, and will use the original data
that was associated with that revision, discarding any data changes that were done
by the latest revision. As an exception, data which the snap explicitly chooses to share
across revisions is not touched by the rollback process.
```